src/cpu/cpu.cpp

```diff
@@ -3104,19 +3109,26 @@ class Cpu final {
 		} else if (cpu_type == "486_prefetch") {
 			CPU_ArchitectureType = ArchitectureType::Intel486NewSlow;
 
-			if (cpu_core == "normal") {
+			if (cpu_core == "auto") {
 				cpudecoder = &CPU_Core_Prefetch_Run;
 
-				CPU_PrefetchQueueSize = 32;
-
-			} else if (cpu_core == "auto") {
-				cpudecoder = &CPU_Core_Prefetch_Run;
-
-				CPU_PrefetchQueueSize           = 32;
+				CPU_PrefetchQueueSize         = 32;
 				auto_determine_mode.auto_core = false;
```

Comment:
Same logic here.  Is this is because you're explicitly setting `cpudecoder`?  I'm not going to read through all the logic here.  Just wanted to point it out in case this is a bug.

Reply:
As above, and thanks for flagging it. It's best to double-check such suspicious looking things, no doubt 😄